macros/tuva_invocations/create_tuva_invocations_table.sql

```diff
@@ -0,0 +1,52 @@
+{% macro create_tuva_invocations_table() %}
```

Comment:
Just a thought, but it might be cleaner to group all of these related macros into one macro file.